spec/presenters/presenter_spec.rb

```diff
@@ -20,4 +20,19 @@
       expect(subject.salary_frequency_options).to eq(expected_result)
     end
   end
+
+  describe '#formatted_currency' do
+    let(:number) { 5_876 }
+    context 'when a precision is provided' do
```

Comment:
I would have expect the precision to the provided as part of the the test setup but there's nothing here.